linkup-cli/src/status.rs

```diff
@@ -140,7 +140,13 @@ fn format_state_domains(state: &LocalState) -> Vec<String> {
 
     return filtered_domains
         .iter()
-        .map(|d| format!("{}.{}", state.linkup.session_name.clone(), d.clone()))
+        .map(|d| {
+            format!(
+                "https://{}.{}",
+                state.linkup.session_name.clone(),
+                d.clone()
+            )
+        })
```

Comment:
**_nit:_**
Since this is being touched, maybe we can improve readability here 👀 
```suggestion
        .map(|domain| {
            format!(
                "https://{}.{}",
                state.linkup.session_name.clone(),
                domain.clone()
            )
        })
```